broker/src/scheduler/active_calls_gc.erl

```diff
@@ -24,9 +29,36 @@ handle_call(_Request, _From, State) ->
 handle_cast(_Msg, State) ->
   {noreply, State}.
 
+notify_failed_to_callback_url(Call) ->
+  case Call#call_log.callback_url of
+    undefined -> ok;
+    <<>> -> ok;
+    Url ->
+      CallSid = util:to_string(Call#call_log.id),
+      SessionVars = case Call#call_log.js_context of
+        "" -> [];
+        undefined -> [];
+        _ -> session:session_vars(Call#call_log.js_context)
+      end,
+      spawn(fun() ->
```

Comment:
Isn't there any other function we could reuse that takes care of generating a callback URL, its parameters, and calling it? It smells like there's a big part of this function that should already be implemented for the other callbacks we have.

Reply:
The function that handles this is within `session` and is called `notify_status_to_callback_url`, but needs a session to make the callback. Here I'm reusing the minimum of code without the need for a session (that we don't "have" in the GC, since we are killing calls that would not have one). 

Not sure what is more clear/convenient, accepting opinions.

Reply:
I guess we could refactor out the two tasks those functions are doing: choosing the parameters, then executing the callback. But it's not required 👌

Reply:
In that case adding a `TODO: remove duplication with XXX would be nice` in both locations. If we ever change one we'll be reminded to change the other _or_ to refactor into a single function :)